src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverterOfT.cs

```diff
@@ -64,6 +77,11 @@ internal sealed override JsonParameterInfo CreateJsonParameterInfo()
             return new JsonParameterInfo<T>();
         }
 
+        internal sealed override JsonConverter<TargetType> CreateCastingConverter<TargetType>()
+        {
+            return new CastingConverter<TargetType, T>(this);
```

Comment:
We should follow generic parameter naming convention:

```suggestion
        internal sealed override JsonConverter<TTarget> CreateCastingConverter<TTarget>()
        {
            return new CastingConverter<TTarget, T>(this);
```